base/docs/basedocs.jl

```diff
@@ -1888,4 +1888,35 @@ The syntax `a.b = c` calls `setproperty!(a, :b, c)`.
 """
 Base.setproperty!
 
+"""
+    StridedArray{T, N}
+
+An `N` dimensional *strided* array with elements of type `T`. A strided array may *not* be contiguous
```

Comment:
How about s/may/might?  "May not" sounds like it must not be contiguous.

I think it'd be good to introduce what strides are first.  This is a little tricky.  Here's one shot:

> A union of common builtin `N`-dimensional array types with elements of type `T` that follow the [strided array interface](https://docs.julialang.org/en/latest/manual/interfaces/#man-interface-strided-arrays-1)
>
> If `A isa StridedArray`, then its entries are stored in memory with fixed offsets (or "strides") such that incrementing an index along dimension `d` jumps ahead in memory by `strides(A, d)` slots.  Strided arrays are particularly important because they can sometimes be passed directly as pointers to foreign language libraries like BLAS.

Also note that strides of 1 isn't sufficient for contiguousness.  The condition is: `strides(A, 1) == 1 && strides(A)[2:end] == size(A)[1:end-1]`, but I don't think we need to talk about contiguousness here.